kong/runloop/handler.lua

```diff
@@ -204,6 +211,16 @@ local function csv(s)
 end
 
 
+local function try_incr_rebuilds_counter(key)
```

Comment:
Actually, second thought, I think it might be advantageous to remove this function and just do the incremental in line. It will probably be the same amount of lines, but the error message can be more specific.

Using a seprate function that forces the key to be either `ROUTERS_REBUILD_COUNTER_KEY` or `PLUGINS_REBUILD_COUNTER_KEY` seems to not save any code.